docs/Operators.md

```diff
@@ -687,12 +693,14 @@
    data into the output tensor Y for further processing.
   * **attribute**:
     <dl>
+      <dt>auto_pad</dt>
+      <dd>If specified, it will overrides pads attribute values.auto_pad takes VALID or SAME, VALID means don't pad and same meanspad by matching the input spatial size.</dd>
       <dt>dilations</dt>
       <dd>Dilation along each axis, 1 means no dilation.</dd>
       <dt>kernel_shape</dt>
       <dd>The size of the kernel along each axis.</dd>
       <dt>pads</dt>
-      <dd>Padding along each axis, can take the value 0 (False) or non 0 (True)</dd>
+      <dd>Padding along each axis, can take any value greater than or equal to 0.The number represent the added pixels in the lower and upper part of thecorresponding  axis.</dd>
```

Comment:
Typo: "thecorresponding"

Could you also clearly state that the added pixels will be padded as 0?